x-pack/plugins/cases/server/client/cases/utils.test.ts

```diff
@@ -644,22 +669,6 @@ describe('utils', () => {
       });
     });
 
-    it('throws error if connector is not supported', async () => {
```

Comment:
why we deleting this one?

Reply:
So I noticed that test is not valid anymore. The functionality changed in a previous PR. Although the test passes I get the following error:

```
(node:27107) UnhandledPromiseRejectionWarning: Error: expect(received).toEqual(expected) // deep equality

Expected: [Error: Invalid external service]
Received: [Error: Retrieving Incident by id external-id from not-supported failed with exception: Error: exception]
(Use `node --trace-warnings ...` to show where the warning was created)
(node:27107) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 146)
(node:27107) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```